Adds a coordinate plotter map to the application.

blafri868 opened the pull request:
The map allows you to plot the cordinates of a circle relative to the map using a two sliders, one for the X axis and one for the y axis.

blafri868 committed:
Adds a coordinate plotter map to the application.

The map allows you to plot the cordinates of a circle relative to the
map using a two sliders, one for the X axis and one for the y axis.

Vishnu96jm commented on app/src/main/java/com/kodeco/android/coordplot/Map.kt:
```diff
@@ -0,0 +1,44 @@
+import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
+
+@Composable
+fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
```
It's nice that you have created individual files for each composable.

Vishnu96jm commented on app/src/main/java/com/kodeco/android/coordplot/PlotSurface.kt:
```diff
@@ -0,0 +1,57 @@
+            Map(
+                xPercent = xPercentage,
+                yPercent = yPercentage,
+                modifier = Modifier.padding(bottom = 30.dp)
```
```suggestion
                modifier = Modifier.padding(bottom = 30.dp)
```
Good to see that you have added padding bottom of the box. Also try considering adding overall padding to the Column composable of PlotSurface(), so that there is enough space to move the sliders along the edges. :]

Column(
        horizontalAlignment = Alignment.CenterHorizontally,
        verticalArrangement = Arrangement.Center,
        modifier = Modifier
            .fillMaxSize()
            .padding(16.dp)

blafri868 replied:
Will do thanks for the feedback

blafri868 committed:
Adds padding to screen so moving slider is easier

- Padding is added around the plot surface content so that moving the
sliders completly to the left and right is easier.

- Fixes the problem of some of the ball moving off the map when the
  slider is at 0% or 100%
